lib/providers/shuttle_data.dart

```diff
@@ -52,4 +38,393 @@ class ShuttleData extends ChangeNotifier {
+      "1330",
+      "1405",
+      "1420",
+      "1440",
+      "1515",
+      "1530",
+      "1550",
+      "1625",
+      "1640",
+      "1700",
+      "1805",
+      "1840",
+      "1915",
+      "1950"
+    ]
+  },
+  "sgw": {
+    "1":[
+      "745",
+      "805",
+      "820",
+      "835",
+      "855",
+      "910",
+      "930",
+      "945",
+      "1005",
+      "1020",
+      "1055",
+      "1110",
+      "1145",
+      "1215",
+      "1245",
+      "1315",
+      "1345",
+      "1415",
+      "1445",
+      "1515",
+      "1545",
+      "1615",
+      "1645",
+      "1715",
+      "1745",
+      "1815",
+      "1845",
+      "1915",
+      "1945",
+      "2000",
+      "2010",
+      "2030",
+      "2100",
+      "2125",
+      "2145",
+      "2200",
+      "2230",
+      "2300"
+    ],
+    "2":[
+      "745",
+      "805",
+      "820",
+      "835",
+      "855",
+      "910",
+      "930",
+      "945",
+      "1005",
+      "1020",
+      "1055",
+      "1110",
+      "1145",
+      "1215",
+      "1245",
+      "1315",
+      "1345",
+      "1415",
+      "1445",
+      "1515",
+      "1545",
+      "1615",
+      "1645",
+      "1715",
+      "1745",
+      "1815",
+      "1845",
+      "1915",
+      "1945",
+      "2000",
+      "2010",
+      "2030",
+      "2100",
+      "2125",
+      "2145",
+      "2200",
+      "2230",
+      "2300"
+    ],
+    "3":[
+      "745",
+      "805",
+      "820",
+      "835",
+      "855",
+      "910",
+      "930",
+      "945",
+      "1005",
+      "1020",
+      "1055",
+      "1110",
+      "1145",
+      "1215",
+      "1245",
+      "1315",
+      "1345",
+      "1415",
+      "1445",
+      "1515",
+      "1545",
+      "1615",
+      "1645",
+      "1715",
+      "1745",
+      "1815",
+      "1845",
+      "1915",
+      "1945",
+      "2000",
+      "2010",
+      "2030",
+      "2100",
+      "2125",
+      "2145",
+      "2200",
+      "2230",
+      "2300"
+    ],
+    "4":[
+      "745",
+      "805",
+      "820",
+      "835",
+      "855",
+      "910",
+      "930",
+      "945",
+      "1005",
+      "1020",
+      "1055",
+      "1110",
+      "1145",
+      "1215",
+      "1245",
+      "1315",
+      "1345",
+      "1415",
+      "1445",
+      "1515",
+      "1545",
+      "1615",
+      "1645",
+      "1715",
+      "1745",
+      "1815",
+      "1845",
+      "1915",
+      "1945",
+      "2000",
+      "2010",
+      "2030",
+      "2100",
+      "2125",
+      "2145",
+      "2200",
+      "2230",
+      "2300"
+    ],
+    "5":[
+      "745",
+      "820",
+      "855",
+      "930",
+      "945",
+      "1005",
+      "1055",
+      "1110",
+      "1145",
+      "1205",
+      "1220",
+      "1255",
+      "1330",
+      "1335",
+      "1405",
+      "1440",
+      "1455",
+      "1515",
+      "1550",
+      "1605",
+      "1625",
+      "1715",
+      "1730",
+      "1805",
+      "1840",
+      "1915",
+      "1950"
+    ]
+  }
+}
+    ''');
+  }
```

Comment:
Okay now I'm convinced there were some mismerged conflicts. We explicitly tried to pull out this to an actual json file instead of dumping the string here. Double check any other shuttle related changes you might be overwriting. 

might want to merge current dev into your branch to make sure everything is kosher.